tests/test_coreg.py

```diff
@@ -379,7 +380,7 @@ def test_z_scale_corr(self):
         zcorr.fit(self.ref.data, scaled_dem)
 
         # Apply the correction
-        unscaled_dem = zcorr.apply(scaled_dem, None)
+        unscaled_dem = zcorr.apply(scaled_dem, self.ref.transform)
```

Comment:
I'm not sure I get why those needed changing?

Reply:
`Raster.apply()` now raises an error if an array is provided but no transform. This makes the tools perfectly consistent (there was no previous mechanism to validate that a transform was given, so the error messages were very confusing). In this particular case, it's a bit redundant, but I think that's an okay sacrifice for consistency! If you and @adehecq disagree, I'm sure we can find another way.

Reply:
Sounds ok to me